nextcloudappstore/core/templates/account/signup.html

```diff
@@ -0,0 +1,42 @@
+{% extends "account/base.html" %}
+{% load socialaccount %}
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Signup" %}{% endblock %}
+
+{% block content %}
+<h1>{% trans "Sign Up" %}</h1>
+
+<p>{% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">log in</a>.{% endblocktrans %}</p>
```

Comment:
Yes, there is account_login which is supplied by the allauth.account.urls.
The variable login_url is supplied by the login view allauth.account.views.LoginView and generate using reverse.